### ISSUE TITLE
About the singing voice conversion model

### ISSUE THREAD

**Issue by Alidaling** [Plachtaa/seed-vc#55]
May I ask what training data were used for the singing voice conversion model? Was only M4Singer used? Additionally, why is the sampling rate of the singing voice conversion on the demo only 22.05k, while the singing voice conversion model mentioned in the paper uses 44.1k?

**Comment by Plachtaa** [comment]
44.1kHz is just following previous SVC work, as it seems music is supposed to come with high sampling rate. Dataset original sampling rate varies from 16kHz~48kHz and covers a wide range of distribution, m4singer is part of it but not all

**Comment by yiwei0730** [comment]
Can I ask about the training data for the SVC model? According to the content of the paper, is the training data is emilia+M4singer? or there have other music dataset? Approximately how many hours does it take?
